packages/react-accounts/react-accounts.ts

```diff
@@ -0,0 +1,29 @@
+import { Meteor } from 'meteor/meteor'
+import { Tracker } from 'meteor/tracker'
+import { useState, useEffect } from 'react'
+
+export const useUserId = () => {
+  const [userId, setUserId] = useState(Meteor.userId())
```

Comment:
Maybe we could use here the Meteor state if the package is available?

Reply:
Meteor State is probably not necessary here since that data is so cheap to recreate, and Meteor.userId is fairly  deterministic.